### PR TITLE
preface + introduction

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: whitepaper/chapters/introduction.tex [hunk 1]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
+NIS1 was a worthy entry into the blockchain landscape and \codenamespace is an even worthier evolution of it.
+This is not the end but a new beginning.
+There is more work to be done.
+Stay tuned for more things to come.
+
+\subsection{Variants}
+
+\codenamespace supports compile time substitution of its primary hash algorithm that produces a 32-byte value from input data.
+For conciseness, the rest of this document will refer to this hash assuming its default setting (SHA3).
+Currently, the following hash algorithms are supported:
```
Comment (gimre-xymcity) [inline whitepaper/chapters/introduction.tex:43]: how to refer to sha and/or keccak with things like secret lock?

**Review by gimre-xymcity — APPROVED** [review]

File: whitepaper/chapters/introduction.tex [hunk 2]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
-He'd say hello and introduce himself, but most of the cats turned a deaf ear, pretending they couldn't hear him, or stare right through him.
-}{Haruki Murakami}
+From the ashes, a fire shall be woken, A light from the shadows shall spring...
+}{J.R.R. Tolkien}
 
-\nemchapterfirstletter{N}{EM}, in its most basic form, is a crypto currency that is built on block chain technology.
+\nemchapterfirstletter{T}{rustless}, high-performance, layered-architecture, blockchain-based DLT protocol - these are the first principles that influenced the development of \codename.
+
+While other DLT protocols were considered, including DAG and dBFT, blockchain was quickly chosen as the protocol most true to the ideal of trustlessness.
+Any node can download complete copy of the chain and can independently verify it at all times.
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex]: download *a/the* complete

File: whitepaper/chapters/introduction.tex [hunk 3]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
+Nodes with sufficient harvesting power can always create blocks and never need to rely on a leader.
+These choices neccesarily sacrifice some throughput relative to other protocols, but they seem most consistent with the philosophical underpinnings of Bitcoin.
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex]: necessarily

File: whitepaper/chapters/introduction.tex [hunk 4]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
+
+As part of a focus on trustlessnes, the following features were added relative to NEM:
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex]: trustlessness

File: whitepaper/chapters/introduction.tex [hunk 5]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
 \label{sec:introduction}
 
 \nemquote{
-He'd say hello and introduce himself, but most of the cats turned a deaf ear, pretending they couldn't hear him, or stare right through him.
-}{Haruki Murakami}
+From the ashes, a fire shall be woken, A light from the shadows shall spring...
+}{J.R.R. Tolkien}
 
-\nemchapterfirstletter{N}{EM}, in its most basic form, is a crypto currency that is built on block chain technology.
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex:8]: block chain or blockchain, we should be consistent

File: whitepaper/chapters/introduction.tex [hunk 6]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
+In \codename, there is a single server executable that can be customized by loading different plugins (for transaction support) and extensions (for functionality). There are three primary configurations (per network):
+
+\begin{enumerate}
+\item peer: These nodes are the backbone of the network and create new blocks.
+\item api: These nodes store data in a monogo database for easier querying and can be used in combination with a NodeJS REST server.
+\item dual: These nodes perform the functions of both peer and api nodes.
+\end{enumerate}
+
+A strong network will typically have a large number of peer nodes and enough api nodes to support incoming client requests.
+Allowing the composition of nodes to vary dynamically based on real needs, should lead to a more globally resource-optimized network.
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex:31]: is , correct?

File: whitepaper/chapters/introduction.tex [hunk 7]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
+
+Basing the core block and transaction pipelines on the disruptor pattern - and using parallel processing wherever processing - allows high rates of transactions per second relative to a typical blockchain protocol.
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex]: `parallel processing wherever processing` => `parallel processing whenever possible`?
(processing is not always parallel)

File: whitepaper/chapters/introduction.tex [hunk 8]
```diff
@@ -2,7 +2,47 @@ \section{Introduction}
...
+
+NIS1 was a worthy entry into the blockchain landscape and \codenamespace is an even worthier evolution of it.
```
Comment (BloodyRookie) [inline whitepaper/chapters/introduction.tex:35]: what is the difference between \codename and \codenamespace? Both seems to be resolved to `Catapult`

File: whitepaper/preface.tex [hunk 9]
```diff
@@ -5,9 +5,24 @@ \section*{Preface}
...
+Although there was some initial discussion on what to build, we quickly decided to create something new from scratch.
+We wanted to challenge ourselves and see if we could build something useful.
+As a result of lots of effort - mostly nights and weekends - this culminated in the release of NIS1 mainnet in March 2015.
+We were pleased with what we built, but knew we took a few shortcuts, and continued improving it.
+Eventually, we came to the realization that the original solution would need a rearchitecture to fix some central performance bottlenecks and allow faster innovation in the future.
+
+We are grateful to TechBureau who provided support for us to build a completely new chain from scratch - \codename.
+We are hopeful that this fixes many of the problems inherent in NIS1 and provides a solid foundation for future improvements and enhancements.
+Our mandate was to build a high performance *blockchain* - not a DAG or dBFT based system.
+In this, we think we succeded.
```
Comment (BloodyRookie) [inline whitepaper/preface.tex]: think, we ?

**Review by BloodyRookie — CHANGES_REQUESTED** [review]

File: whitepaper/chapters/introduction.tex [hunk 10]
```diff
@@ -22,14 +24,14 @@ \section{Introduction}
 
 \begin{enumerate}
 \item peer: These nodes are the backbone of the network and create new blocks.
-\item api: These nodes store data in a monogo database for easier querying and can be used in combination with a NodeJS REST server.
+\item api: These nodes store data in a mongo database for easier querying and can be used in combination with a NodeJS REST server.
```
Comment (dgarcia360) [inline whitepaper/chapters/introduction.tex:26]: mongo database -> MongoDB document database

**Pull request merged**